src/main/java/org/codeforamerica/shiba/output/documentfieldpreparers/DocumentFieldPreparers.java

```diff
@@ -27,17 +27,13 @@ public DocumentFieldPreparers(List<DocumentFieldPreparer> preparers) {
   public List<DocumentField> prepareDocumentFields(Application application, Document document,
       Recipient recipient) {
 
-    SubworkflowIterationScopeTracker scopeTracker = new SubworkflowIterationScopeTracker();
-
     // Add default fields
     List<DocumentField> fields = new ArrayList<>(getDefaultFields(application));
 
     // Run all the preparers
     preparers.forEach(preparer -> {
       try {
-        fields.addAll(
-            preparer.prepareDocumentFields(application, document, recipient, scopeTracker)
-        );
+        fields.addAll(preparer.prepareDocumentFields(application, document, recipient, null));
```

Comment:
Can we just remove `scopeTracker` from the signature of `prepareDocumentFields` entirely?

Reply:
We can! I was thinking it would be easier to code-review if it was left in - I'll remove it now that it's been reviewed 👍

Reply:
It definitely made it easier—far less noise! Thanks for that :)